src/ildasm/dasm.cpp

```diff
@@ -6688,11 +6708,11 @@ void DumpEATEntriesWrapper(void* GUICookie,
     IMAGE_NT_HEADERS32 *pNTHeader32, IMAGE_OPTIONAL_HEADER32 *pOptHeader32,
     IMAGE_NT_HEADERS64 *pNTHeader64, IMAGE_OPTIONAL_HEADER64 *pOptHeader64)
 {
-    WIN_PAL_CPP_TRY
+    PAL_CPP_TRY
```

Comment:
It would be nice to delete WIN_PAL_CPP_TRY macros from palclr_win.h as well since ildasm were the only placed where they was used, and it is going away with your change.